test/cql-pytest/test_shedding.py

```diff
@@ -69,8 +67,10 @@ def get_protocol_errors():
 
     # disable compression since we rely on specific request size
     with disable_compression():
-        # separate session is needed due to the cql_test_connection fixture,
-        # which checks that the session is not broken at the end of the test
+        # Big request causes the entire connection to be closed (see #8800 for the reasons).
+        # For some reason the driver doesn't try to reopen the connection to the same host,
+        # so we end up with a broken session and cql_test_connection fixture fails the test.
+        # We create a separate session to make the test work.
```

Comment:
Next time, please conflate such a patch that fixes a comment with the earlier patch that introduces the comment in the first place. But it's not important enough to require a new version. Let's just wait for the CI and merge this version.

Reply:
What does 'conflate' mean here? The original patch is already merged, we can't change it.

Reply:
Sorry, I was under the mistaken impression that this specific comment was introduced in the first patch in this series. But now I see I was wrong.